### ISSUE TITLE
Convention mode

### ISSUE THREAD

**Issue by jnackmclain** [hmxmilohax/rock-band-3-deluxe#671]
A convention/demo style mode switch for locking down specific environments for parties/gatherings/bars/conventions/anything.

Idea being you can have your own set of settings, saved separately to dx_demo_settings.dta or something.

Set modes, setup if you want the flow of the game to be locked into quickplay, or play a show.

Don't allow going back or going to different places.
so, dont allow going back from song select to main menu. dont allow going to practice from the more info screen

Restrict pause menu and overshell options. Dont allow going into options/goals/characters/etc.
Only serve navigational options such as pause/back to music library/skip song/drop out

Allow an overshell button combination cheat to disable the mode, this would act to unlock all options for tweaking the experience by the host

Restrict "replay" on endgame, always proceed forward

More ideas should be put on this feature request

**Comment by carlmylo** [comment]
Would be cool if this was a build system option. Being able to lock the menu icon to a specific one that would promote the event venue/RB3DX logo would be nice along with altering the bottom menu text.

**Comment by jnackmclain** [comment]
I don't think it needs its own build system as the entire mode can be enabled via overshell.
Once enabled via overshell, we can have the game reboot in this state until it is turned off by the host button combination

**Comment by LinosM** [comment]
Disable writing dx_values.dta

People often change their difficulty mid song, and saving the difficulty to file causes the game to stutter for a second on hardware

**Comment by DarkRTA** [comment]
imo we shouldn't be writing that file mid-song regardless. opening a separate issue for that

**Comment by SonicSBL** [comment]
It looks like this feature has been implemented for a while now, but I can't find it anywhere in the game. What does one need to do to activate it?

I'm on ps3 if that matters.

**Comment by hubjit23** [comment]
I too am curious about this. How?
